PD-1399 / 24.10 / PD-1399 Terminating an Active Running Job Snippet (#3122)
truenas/documentation: static/includes/TerminateActiveRunningJob.md

&NewLine;

{{< hint type=warning title="Warning" >}}

> Certain jobs, such as debugs, cannot be aborted once they have started. To abort such jobs, advanced users can use the API; however, this is not generally recommended. In the majority of instances, it is more reliable to wait for the job to finish without interference.

Reviewer: Don't use the word once. Use either after or when based on the context.
```suggestion
Certain jobs, such as debugs, cannot be aborted after they start. Advanced users can use the API to abort such job; however, this is not generally recommended.
In the majority of instances, it is more reliable to wait for the job to finish without interference.
```
